apps/dotcms-ui/src/app/view/components/dot-contentlet-editor/components/dot-contentlet-wrapper/dot-contentlet-wrapper.component.ts

```diff
@@ -83,6 +85,13 @@ export class DotContentletWrapperComponent {
                 },
                 'edit-contentlet-loaded': (e: CustomEvent) => {
                     this.header = e.detail.data.contentType;
+                    this.titleService.setTitle(
+                        `${
+                            e.detail.data.pageTitle
+                                ? e.detail.data.pageTitle + ' - '
+                                : this.titleService.getTitle()
+                        } ${this.titleService.getTitle().split(' - ')[1]}`
```

Comment:
Can't we just hardcode this part `${this.titleService.getTitle().split(' - ')[1]}`? Which I'm not mistaken is "dotCMS Platform"

Reply:
are you sure? by doing what I implemented, we keep what ever we have set on the main `index.html` file https://github.com/dotCMS/core-web/blob/397211cc9f4f6b3f657404413f8dc4f72bba5736/apps/dotcms-ui/src/index.html#L10

so any future change to that title is done on a single place